### ISSUE TITLE
Table: enable interacting with rows, and navigating them, using keyboard

### ISSUE THREAD

**Issue by Kiarokh** [Lundalogik/lime-elements#2386]
## New feature motivation
Currently, users can only click on a row in the table, to activate it. There is no way to use the <kbd>⇥ Tab</kbd> key to select a row. It's not possible to activate a row, using `Enter` or `Space` keys neither.

Luckily, the pagination buttons are focusable already, using tab.
<img width="412" alt="image" src="https://github.com/Lundalogik/lime-elements/assets/35954987/f03c10fa-a644-4452-bf07-235ecce2bc41">

## New feature description
1. User should be able to `focus` a row using<kbd>⇥ Tab</kbd>, and then use <kbd>↑</kbd> and <kbd>↓</kbd> keys to switch focus. 
2. The next press of the Tab key should focus on the next focusable element (If there is something inside the row, the focus should go there. If next focusable element is outside in the DOM, the focus should go there).
3. pressing <kbd>↩ Return</kbd> or <kbd>SPACE</kbd> on the focused row should do the same thing that `onClick` does.

## New feature implementation

- [ ] Should we first upgrade to the newest Tabulator instead? 🤔 Maybe they have fixed these issues.
- [ ] enable tabbing, by adding `tabindex="0"` to the row `div`s
- [ ] add code for `handleKeyDown`
- [ ] add proper styles for focused state, etc…

**Comment by Kiarokh** [comment]
To make this to properly work in the Web Client, we also need to fix this issue too https://github.com/Lundalogik/crm-feature/issues/1294

The reason is that when we close the floating card, the previously focused element (table row) will be lost, and the user has to start all over again.
